lib/begin.go

```diff
@@ -95,9 +119,13 @@ func (a *ACBuild) Begin(start string, insecure bool) (err error) {
 			case finfo.IsDir():
 				return a.beginFromLocalDirectory(start)
 			default:
-				return a.beginFromLocalImage(start)
+				return a.beginFromLocalImage(start, mode)
 			}
 		} else {
+			if mode == BuildModeOCI {
+				// TODO: fix this!
+				return fmt.Errorf("cannot start from remote OCI images currently")
```

Comment:
we'll should likely start using containers/image for this sort of images fetching. Eventually you can implement an appc:// transport in containers image and use something like "docker://busybox appc://busybox" to make the conversion directly.